src/utils/wrap-number.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * A function to allow wrapping from the beginning of a series of numbers to the end, and from the end of the series to the beginning.
+ * @param {number} numberToNormalize - The number to check
```

Comment:
```suggestion
 * @param {number} numberToWrap - The number to check
```

We want this JSDoc to match